<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.0" language="de">
<context>
    <name>ImportProject</name>
    <message>
        <location filename="base/file/Ui_importproject.py" line="127"/>
        <source>Import Project</source>
        <translation>Projekt importieren</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="128"/>
        <source>Import data </source>
        <translation>Daten importieren</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="129"/>
        <source>Input file: </source>
        <translation>Input Datei:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="130"/>
        <source>Browse</source>
        <translation>Durchsuchen</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="131"/>
        <source>Project name: </source>
        <translation>Projektname:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="132"/>
        <source>Check</source>
        <translation>Prüfen</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="133"/>
        <source>Date: </source>
        <translation>Datum:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="134"/>
        <source>Reference frame: </source>
        <translation>Bezugsrahmen:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="135"/>
        <source>Application module: </source>
        <translation>Fachschale:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="136"/>
        <source>Ili model name: </source>
        <translation>Modellname:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="137"/>
        <source>Output:</source>
        <translation>Output:</translation>
    </message>
    <message>
        <location filename="base/file/Ui_importproject.py" line="138"/>
        <source>Notes:</source>
        <translation type="unfinished">Bemerkungen:</translation>
    </message>
</context>
<context>
    <name>ImportProjectDialog</name>
    <message>
        <location filename="base/file/doImportProject.py" line="57"/>
        <source>Error while reading modules.json.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="82"/>
        <source>Could not open database: </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="91"/>
        <source>Project name already in use.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="93"/>
        <source>Project name is valid.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="99"/>
        <source>Could not check. Database port is not set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="151"/>
        <source>Choose interlis transfer file</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="168"/>
        <source>Notes are too big (more than 10000 characters).</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="187"/>
        <source>Projects database not found: </source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="191"/>
        <source>No Interlis transfer file set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="195"/>
        <source>No Interlis model name set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="199"/>
        <source>No database schema set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="203"/>
        <source>No application module chosen.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="207"/>
        <source>Missing database parameters.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="211"/>
        <source>No root directory for projects set.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="215"/>
        <source>No projects database found. Will create one in the project root directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="218"/>
        <source>No jar file set for import.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="266"/>
        <source>Import process not sucessfully finished. Could not update projects database.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="274"/>
        <source>Import process not sucessfully finished.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="280"/>
        <source>Import process finished.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="282"/>
        <source>Import process not sucessfully finished. Could not create project directory.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="341"/>
        <source>Error while creating properties file.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="360"/>
        <source>Could not open projects database.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="base/file/doImportProject.py" line="381"/>
        <source>Error while updating projects database.</source>
        <translation type="unfinished"></translation>
    </message>
</context>
<context>
    <name>VeriSO</name>
    <message>
        <location filename="veriso.py" line="279"/>
        <source>&amp;VeriSO</source>
        <translation></translation>
    </message>
    <message>
        <location filename="veriso.py" line="148"/>
        <source>VeriSO</source>
        <translation></translation>
    </message>
    <message>
        <location filename="veriso.py" line="164"/>
        <source>Projects</source>
        <translation>Projekte</translation>
    </message>
    <message>
        <location filename="veriso.py" line="172"/>
        <source>File</source>
        <translation>Datei</translation>
    </message>
    <message>
        <location filename="veriso.py" line="174"/>
        <source>Import project</source>
        <translation>Projekt importieren</translation>
    </message>
    <message>
        <location filename="veriso.py" line="176"/>
        <source>Delete project</source>
        <translation>Projekt löschen?</translation>
    </message>
    <message>
        <location filename="veriso.py" line="186"/>
        <source>Settings</source>
        <translation>Einstellungen</translation>
    </message>
    <message>
        <location filename="veriso.py" line="188"/>
        <source>Options</source>
        <translation>Optionen</translation>
    </message>
    <message>
        <location filename="veriso.py" line="275"/>
        <source>Error while loading application module: </source>
        <translation></translation>
    </message>
</context>
<context>
    <name>VeriSO_EE</name>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="48"/>
        <source>Checks</source>
        <translation>Tests</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="108"/>
        <source>Baselayer</source>
        <translation>Basislayer</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="153"/>
        <source>Tables</source>
        <translation>Tabelle</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="179"/>
        <source>Load Topic</source>
        <translation>Topic laden</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="207"/>
        <source>Defects</source>
        <translation>Mängel</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="209"/>
        <source>Load defects layer</source>
        <translation>Mängellayer laden</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/applicationmodule.py" line="213"/>
        <source>Export defects layer</source>
        <translation>Mängellayer exportieren</translation>
    </message>
</context>
<context>
    <name>VeriSO_EE_FP3</name>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="41"/>
        <source>FixpunkteKategorie3</source>
        <translation>FixunkteKategorie3</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="46"/>
        <source>Toleranzstufen</source>
        <translation>Toleranzstufen</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="80"/>
        <source>LFP3</source>
        <translation>LFP3</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="94"/>
        <source>LFP3 ausserhalb Gemeinde</source>
        <translation>LFP3 ausserhalb Gemeinde</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="120"/>
        <source>Gemeindegrenze</source>
        <translation>Gemeindegrenze</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="36"/>
        <source>project_id not set</source>
        <translation>Parameter _project_id_ nicht gesetzt.</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="67"/>
        <source>LFP3 NachfÃ¼hrung</source>
        <translation>LFP3 Nachführung</translation>
    </message>
    <message>
        <location filename="modules/veriso_ee/complexchecks/fp3.py" line="109"/>
        <source>LFP1 + LFP2 Schweiz</source>
        <translation></translation>
    </message>
</context>
</TS>
